vm/vmgs/vmgstool/src/main.rs

```diff
@@ -1145,391 +1146,358 @@ mod tests {
+        //File size must be % 512 to be valid, should produce error and file should not be created
+        let result = test_vmgs_create(&path, Some(65537), false, None).await;
+        assert!(result.is_err());
+        assert!(!path.exists());
     }
 
-    #[test]
-    fn test_forcecreate() {
-        DefaultPool::run_with(|_| async move {
-            let (_dir, path) = new_path();
-            let result = test_vmgs_create(&path, Some(4194304), false, None).await;
-            assert!(result.is_ok());
-            // Recreating file should fail without force create flag
-            let result = test_vmgs_create(&path, Some(4194304), false, None).await;
-            assert!(result.is_err());
-            // Should be able to resize the file when force create is passed in
-            let result = test_vmgs_create(&path, Some(8388608), true, None).await;
-            assert!(result.is_ok());
-        })
+    #[async_test]
+    async fn test_forcecreate() {
+        let (_dir, path) = new_path();
+        let result = test_vmgs_create(&path, Some(4194304), false, None).await;
+        assert!(result.is_ok());
+        // Recreating file should fail without force create flag
+        let result = test_vmgs_create(&path, Some(4194304), false, None).await;
+        assert!(result.is_err());
+        // Should be able to resize the file when force create is passed in
+        let result = test_vmgs_create(&path, Some(8388608), true, None).await;
+        assert!(result.is_ok());
     }
 
     #[cfg(with_encryption)]
-    #[test]
-    fn test_update_encryption_key() {
-        DefaultPool::run_with(|_| async move {
-            let (_dir, path) = new_path();
-            let encryption_key = vec![5; 32];
-            let new_encryption_key = vec![6; 32];
-            let buf_1 = b"123".to_vec();
-
-            test_vmgs_create(
-                &path,
-                None,
-                false,
-                Some((EncryptionAlgorithm::AES_GCM, &encryption_key)),
-            )
-            .await
-            .unwrap();
-
-            {
-                let mut vmgs =
-                    test_vmgs_open(&path, FileDiskFlag::ReadWrite, Some(&encryption_key), false)
-                        .await
-                        .unwrap();
-
-                vmgs_write(&mut vmgs, FileId::BIOS_NVRAM, &buf_1, true, false)
-                    .await
-                    .unwrap();
-            }
-
-            test_vmgs_update_key(
-                &path,
-                EncryptionAlgorithm::AES_GCM,
-                Some(&encryption_key),
-                &new_encryption_key,
-            )
-            .await
-            .unwrap();
-
-            {
-                let mut vmgs =
-                    test_vmgs_open(&path, FileDiskFlag::Read, Some(&new_encryption_key), false)
-                        .await
-                        .unwrap();
-
-                let read_buf = vmgs_read(&mut vmgs, FileId::BIOS_NVRAM, true)
+    #[async_test]
```

Comment:
if git just having a hard time reconsiling this merge, or did we lose something?

Reply:
Looks fine to me? Try ignoring whitespace?

Reply:
yup, ignoring whitespace was the move.

Reply:
That's why I mentioned it in the description ;)